__tests__/__snapshots__/operation.test.ts.snap

```diff
@@ -5,160 +5,53 @@ Array [
   Object {
     "label": "Body Params",
     "schema": Object {
-      "$ref": "#/components/schemas/Pet",
```

Comment:
I refactored the test that this snapshot is part of to dereference the petstore first, so these refs are no longer getting added into compiled JSON Schema.